bundles/org.openhab.binding.modbus.sbc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="modbus"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="ald1Unidirectional">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="serial"/>
+		</supported-bridge-type-refs>
```

Comment:
You may want to consider adding support for the RTU over TCP ( #9435 ) by adding the following here..

```
<bridge-type-ref id="tcp"/>
```

Reply:
I agree. People can use serial/ethernet gateways (passing modbus rtu over tcp) or modbus gateways (converting modbus rtu to modbus tcp) with these devices

Reply:
I wasn't aware this is possible. Thanks!

Reply:
Is adding `tcp` bridge something that will be added here?